playbooks/adhoc/uninstall.yml

```diff
@@ -329,21 +328,16 @@
       dest=/etc/sysconfig/docker
       regexp='(ADD_REGISTRY|BLOCK_REGISTRY|INSECURE_REGISTRY)=.*'
 
-  - name: Detect Docker storage configuration
-    shell: vgs -o name | grep docker
-    register: docker_vg_name
-    failed_when: false
-    changed_when: false
-
-  - name: Wipe out Docker storage contents
```

Comment:
I think the vg should be removed as well, it probably should be done after `/var/lib/docker` is removed

Reply:
Yes, I've been thinking about removing VG a lot, however docker-storage-setup has 3 options:

https://docs.openshift.com/container-platform/3.9/install_config/install/host_preparation.html#configuring-docker-thin-pool
> Option A) Use an additional block device.
> Option B) Use an existing, specified volume group.
> Option C) Use the remaining free space from the volume group where your root file system is located.

If users choose B) or C), removing VG by playbook is troublesome (especially Option C causes critical issue). I guess that `docker-storage setup --reset` does not reset the VG for the same reason.

Reply:
Oh, okay, lets leave it then